checkov/sca_package_2/scanner.py

```diff
@@ -24,10 +28,35 @@ def __init__(self, pbar: ProgressBar | None = None, root_folder: str | Path | No
 
     def scan(self) -> Sequence[dict[str, Any]]:
         """run SCA package scan and poll scan results"""
-        pass
+        self.run_scan()
+        return []
 
     def run_scan(self) -> dict[str, Any]:
-        pass
+        logging.info("Start to scan package files.")
+
+        request_body = {
+            "branch": "",
+            "commit": "",
+            "path": os.path.join(bc_integration.repo_path, '') if bc_integration.repo_path else "",
+            "repoId": bc_integration.repo_id,
+            "id": bc_integration.timestamp,
+            "repositoryId": ""
+        }
+
+        response = request_wrapper(
+            "POST", f"{self._base_url}/api/v1/vulnerabilities/cli/scan",
+            headers=bc_integration.get_default_headers("POST"),
+            json=request_body,
+            should_call_raise_for_status=True
+        )
+
+        response_json = response.json()
+
+        if not response_json["startedSuccessfully"]:
+            logging.info("Failed to run package scanning.")
+            return dict()
+
+        return self.poll_scan_result()
```

Comment:
I think the polling should be under `def scan()`, just return the id here so, it can be used by `poll_scan_result()`